automation/upload_test_artifacts.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -ex
+
+SCP_SRV="grisge.info"
+SCP_BASEDIR="upload"
+SCP_USERNAME="nmstate"
+SCP_KEY="./automation/id_rsa_nmstate"
+BASE_URL="http://grisge.info/nmstate"
+
+if [ -z "$TRAVIS_JOB_ID" ];then
+    echo "Need to be run in traivs-CI, quiting"
```

Comment:
There is a typo: `traivis` instead of `travis`.

Reply:
Done.